fvm/env.go

```diff
@@ -548,6 +548,11 @@ func (e *hostEnv) GetSigningAccounts() ([]runtime.Address, error) {
 	return e.transactionEnv.GetSigningAccounts(), nil
 }
 
+func (e *hostEnv) ImplementationDebugLog(message string) error {
+	e.ctx.Logger.Debug().Msg(fmt.Sprintf("Cadence: %s", message))
```

Comment:
Can `Msgf` be used here?

Reply:
Good idea, changed in f3209c94daa5588d4aee7c7326db3a3c891962e3